examples/transformer.yaml

```diff
@@ -0,0 +1,46 @@
+defaults:
```

Comment:
One more suggestion from my side would be to extend test/config/encoders.yaml to include a model with the new encoder (similar to this config but maybe with a tiny hidden size). Just to make sure future configs won't break this encoder.

Reply:
That's a good idea. I will do this. thanks for the suggestions !!